chainermn/functions/__init__.py

```diff
@@ -0,0 +1,4 @@
+from chainermn.functions.point_to_point_communication import recv  # NOQA
+from chainermn.functions.point_to_point_communication import Recv  # NOQA
+from chainermn.functions.point_to_point_communication import send  # NOQA
+from chainermn.functions.point_to_point_communication import Send  # NOQA
```

Comment:
Do we need to expose `Recv` and `Send` (in addition to `recv` and `send`)? For example, in Chainer, `BatchNormalizationFunction` is not exposed, and only method `batch_normalization` is imported in `__init__.py`.
https://github.com/chainer/chainer/blob/master/chainer/functions/__init__.py#L256
https://github.com/chainer/chainer/blob/master/chainer/functions/normalization/batch_normalization.py